src/popup/components/NameItem.vue

```diff
@@ -252,33 +274,44 @@ export default defineComponent({
       showInput.value = false;
     }
 
-    async function setDefault() {
-      await store.dispatch('names/setDefault', {
-        address: activeAccount.value.address,
-        name: props.name,
+    async function handleSetDefault() {
```

Comment:
I think the body of this function should be moved to `aeNames` composable - similar as it was before done in the Vuex store.

Reply:
Setting the default value for single account address occurs only here so I'd prefer to have it moved here